thirdparty/build-thirdparty.sh

```diff
@@ -256,6 +256,8 @@ build_thrift() {
     fi
 
     make -j $PARALLEL && make install
+    # Create soft link under `/usr/bin`
+    ln -s /var/local/thirdparty/installed/bin/thrift /usr/bin/thrift
```

Comment:
Is there any other options?
Actually this would require `root` permission to do so...
How about adding `/var/local/thirdparty/installed/bin/` to env `PATH`?

Reply:
How about specifying `thrift` path in a POM file, like this:
```xml
 <plugin>
                <groupId>org.apache.thrift.tools</groupId>
                <artifactId>maven-thrift-plugin</artifactId>
                <version>0.1.11</version>
                <configuration>
                    <thriftExecutable>/var/local/thirdparty/installed/bin/thrift</thriftExecutable>
```